features/org.wso2.carbon.appmgt.store.feature/src/main/resources/admin-dashboard/site/themes/default/templates/businessowners-task/js/businessowners.js

```diff
@@ -0,0 +1,274 @@
+                                                  + '</td> <td><a data-target="#entitlement-policy-editor" ' +
+                                                  'data-toggle="modal" data-owner-id="' + obj.businessOwnerId
+                                                  + '" class="owner-edit-button">' +
+                                                  '<i class="icon-edit"></i></a> &nbsp;<a  data-policy-name="'
+                                                  + obj.businessOwnerName +
+                                                  '"  data-owner-id="' + obj.businessOwnerId
+                                                  + '" class="owner-delete-button"><i class="icon-trash"></i>' +
+                                                  '</a></td></tr>');
+        }
+    });
+
+}
+function GetDynamicTextBox(index, key, value) {
+    var id_key = "key-".concat(index);
+    var id_val = "value-".concat(index);
+    var id_btn = "btn-".concat(index);
+    return '<input name = "key" type="text" id="' + id_key + '" value="' + key + '"/>&nbsp &nbsp &nbsp &nbsp' +
+           '<input name="value" type="text" id="' + id_val + '" value="' + value
+           + '"/>&nbsp &nbsp &nbsp &nbsp<button id="' + index
+           + '" class="btn  btn-info" onClick = "removeFields(this.id)">Remove</button>'
+}
+
+function removeFields(index) {
+    var id_key = "#" + "key-".concat(index);
+    var id_val = "#" + "value-".concat(index);
+    var id_btn = "#" + index;
+    $(id_key).val("");
+    $(id_val).val("");
+    $(id_key).hide();
+    $(id_val).hide();
+    $(id_btn).hide();
+}
+
+$(document).on("click", "#btn-owner-add-field", function () {
+    var div = $("<div />");
+    div.html(GetDynamicTextBox(extraFieldCount, "", ""));
+    $("#businessOwnerOther").append(div);
+    extraFieldCount++;
+});
+
+//edit event
+$(document).on("click", ".owner-edit-button", function () {
+    var ownerId = $(this).data("ownerId");
+    editedOwnerId = ownerId;
+    $('#businessOwnerName').val("");
+    $('#businessOwnerEmail').val("");
+    $('#businessOwnerSite').val("");
+    $('#businessOwnerDescription').val("");
+    $('.content-section').show();
+    var section = $('.title-section-edit');
+    var businessOwnerDetails;
+    jQuery('html, body').animate({
+                                     scrollTop: section.offset().top
+                                 }, 1000);
+
+    $.each(businessOwnersArray, function (index, obj) {
+        if (obj != null && obj.businessOwnerId == ownerId) {
+            $('#businessOwnerName').val(obj.businessOwnerName);
+            $('#businessOwnerEmail').val(obj.businessOwnerEmail);
+            $('#businessOwnerSite').val(obj.businessOwnerSite);
+            $('#businessOwnerDescription').val(obj.businessOwnerDescription);
+
+            $.ajax({
+                       url: context + '/apis/businessowners/details/' + obj.businessOwnerId,
```

Comment:
API context should contain "/apis/businessowners/cusotmproperties" rather than "/apis/businessowners/details"